### ISSUE TITLE
Email Invite to Start Up Editors

### ISSUE THREAD

**Issue by rchampieux** [ryerbanta/radian#4]
Dear X,

I hope your summer is going well. I'm writing to ask you to participate in a new project that I'm launching with a team of colleagues from OHSU, Centralia Community College, and Montana State University. The project is described below:

## The Problem

Great data related resources are out there. Many of you—data experts, practitioners, and learners alike—share your data solutions, approaches, and ideas on the open web. However, finding these resources is hard, especially when materials with the potential for broad appeal are only shared with a specialized community. We want to break down disciplinary silos and create a space to share resources, amplify your work, and build an interdisciplinary community.

## Introducing Radian

Radian provides a community-curated hub for sharing resources and discovering ideas about data science, visualization, data management, data policy, and applications of data for the public good. By highlighting and connecting the work you are using and creating in your practice we want to increase the discovery and recognition of data-related contributions on the open web, and foster a cross-disciplinary community built on information sharing.

## Where you come in

We're looking for a group of startup editors to help launch Radian. As an editor, you'll use [PressForward technology](http://pressforward.org/) to promote the awesome content that you make and use in your work, and then you'll add some extra information to make it easier to discover and use by others. You will be recognized as a Radian Editors-at-Large and credited for the posts that you contribute. 

We thought of you because of [your expertise in X, your website, your work with x, your vibrant Twitter presence, etc]. As a community leader, your voice will be invaluable in helping to populate Radian with great content and thoughtful commentary.

I hope you'll agree to be one of Radian's inaugural editors. Let me know if you have questions!

Sara

**Comment by rchampieux** [comment]
@saramannheimer this looks great.  Just a few small changes reflected below and a bit of new content related to timeline. 

Dear X,

I hope your summer is going well. I'm writing to ask you to participate in a new project that I'm launching with a team of colleagues from OHSU, Centralia College, and Montana State University. The project is described below:

## The Problem

Great data related resources are out there. Many of you—data experts, practitioners, and learners alike—share your data solutions, approaches, and ideas on the open web. However, finding these resources is hard, especially when materials with the potential for broad appeal are only shared with a specialized community. We want to break down disciplinary silos and create a space to share resources, amplify your work, and build an interdisciplinary community.

## Introducing Radian

Radian will provide a community-curated hub for sharing resources and discovering ideas about data science, visualization, data management, data policy, and applications of data for the public good. By highlighting and connecting the work you are using and creating in your practice we want to increase the discovery and recognition of data-related contributions on the open web, and foster a cross-disciplinary community built on information sharing.

## Where you come in

We're looking for a group of startup editors to help launch Radian. As an editor, you'll use the [PressForward technology](http://pressforward.org/) to promote the awesome content that you make and use in your work, and then you'll add some extra information to make it easier to discover and use by others. You will be recognized as a Radian Editors-at-Large and credited for the posts that you contribute.  The workflow is straight forward and built to leverage your existing browsing, publishing, and reuse practices. 

We thought of you because of [your expertise in X, your website, your work with x, your vibrant Twitter presence, etc]. As a community leader, your voice will be invaluable in helping to populate Radian with great content and thoughtful commentary.

I hope you'll agree to be one of Radian's inaugural editors.  We plan to launch the site in October, and will be hosting a quick training session for editors in September. Let me know if you have questions!

Sara

**Comment by ryerbanta** [comment]
My institution is just Centralia College.

Overall this is looking great. I think the sections really help its readability. The section that introduces Radian is the only one that may be a little too dense or hard to parse. It's not terrible, but could benefit from some polishing.

**Comment by ryerbanta** [comment]
Drafting to improve language for the Introducing Radian section.

## Introducing Radian

Radian will be a community-curated hub for all things data. Use Radian to discover ideas and share resources about data science, visualization, data management, data policy, and applications of data for the public good. Radian will feature the best work created and used by the cross-disciplinary data community. Highlighting and connecting this work will increase the discovery and recognition of data-related contributions on the open web, and foster community built on information sharing.

**Comment by rchampieux** [comment]
Integrated text, feel free to make it your own for personal outreach.

Dear X,

I hope your summer is going well. I'm writing to ask you to participate in a new project that I'm launching with a team of colleagues from OHSU, Centralia College, and Montana State University. The project is described below:

## The Problem

Great data related resources are out there. Many of you—data experts, practitioners, and learners alike—share your data solutions, approaches, and ideas on the open web. However, finding these resources is hard, especially when materials with the potential for broad appeal are only shared with a specialized community. We want to break down disciplinary silos and create a space to share resources, amplify your work, and build an interdisciplinary community.

## Introducing Radian

Radian will be a community-curated hub for all things data. Use Radian to discover ideas and share resources about data science, visualization, data management, data policy, and applications of data for the public good. Radian will feature the best work created and used by the cross-disciplinary data community. Highlighting and connecting this work will increase the discovery and recognition of data-related contributions on the open web, and foster community built on information sharing.

## Where you come in

We're looking for a group of startup editors to help launch Radian. As an editor, you'll use the [PressForward technology](http://pressforward.org/) to promote the awesome content that you make and use in your work, and then you'll add some extra information to make it easier to discover and use by others. You will be recognized as a Radian Editors-at-Large and credited for the posts that you contribute.  The workflow is straight forward and built to leverage your existing browsing, publishing, and reuse practices. 

We thought of you because of [your expertise in X, your website, your work with x, your vibrant Twitter presence, etc]. As a community leader, your voice will be invaluable in helping to populate Radian with great content and thoughtful commentary.

I hope you'll agree to be one of Radian's inaugural editors.  We plan to launch the site in October, and will be hosting a quick training session for editors in September. Let me know if you have questions!
